### DISCUSSION TITLE
Duplicate Icons after update

### DISCUSSION THREAD

**Discussion by tianchrisy** [Romanitho/Winget-AutoUpdate#258] [Q&A] [answered]
Hey Everyone!

I have successfully deployed this to our test machines and working beautifully.

However, I have discovered an issue with one of our machines.

After updating apps, we would notice duplicate application icon appear on the desktop. Calibre, Anki, and gVIM are some of the applications that are doing this. 

Has anyone experienced this? So far it is only happening to one machine. 
![Winget Duplicate](https://user-images.githubusercontent.com/93405388/210835699-aba70642-a273-407a-b209-bd38779a4099.png)

**Comment by KnifMelti** [comment]
? I don't see any duplicates on that desktop (different names on them)...

**Reply by tianchrisy** [reply]
hmmm true they are different names. but we never actually had those icons on our desktop before. (We did have the application installed just not desktop icons)

**Comment by KnifMelti** [comment, marked as answer]
That's up to the application itself (if it creates shortcuts or not on the desktop).
Maybe you've had someone like myself (**packager**) config the applications before adding them in the Deployment system?
Then that person would have made sure that the shortcuts got deleted/not was added at all just because the application flora would soon fill up the desktop with no way for the user to delete them.

An easy way for solving this is to use the [Custom scripts (Mods feature)](https://github.com/Romanitho/Winget-AutoUpdate#custom-scripts-mods-feature) in **WAU**

As soon as @Romanitho has released a new version of **WAU** my contributions to this are going to be of a big help for non scripters... #250

**Reply by tianchrisy** [reply]
Oh Goodness. I totally missed that part of the Documentation. Thanks! That is very helpful.

**Reply by KnifMelti** [reply]
Because this is developed constantly, the description in my latest PR #257  now looks like this:
https://github.com/KnifMelti/Winget-AutoUpdate/tree/GPO#custom-scripts-mods-feature

**Reply by KnifMelti** [reply]
**List**/**Mods** can be combined with **-ListPath**/**-ModsPath** (read about it under [Advanced installation](https://github.com/KnifMelti/Winget-AutoUpdate/tree/GPO#advanced-installation) - I'm now using an Apache public URL Path on my own domain to deliver mods for all our whitelisted applications (moving to an IIS public URL Path on our domain):
https://www.knifmelti.se/wau/mods/
To get it as it should be you'll have to wait for #257
